Lib/test/test_zipfile/test_core.py

```diff
@@ -645,6 +645,17 @@ def test_add_file_after_2107(self):
             zinfo = zipfp.getinfo(TESTFN)
             self.assertEqual(zinfo.date_time, (2107, 12, 31, 23, 59, 59))
 
+    def test_ZipInfo_compresslevel_property(self):
```

Comment:
`AbstractTestsWithSourceFile` is a wrong place for this test. `ZipInfoTests` is perhaps a correct place.

Reply:
good catch, thanks!